nrf52840.robot

```diff
@@ -0,0 +1,71 @@
+*** Settings ***
+Suite Setup                   Setup
+Suite Teardown                Teardown
+Test Setup                    Reset Emulation
+Test Teardown                 Test Teardown
+Resource                      ${RENODEKEYWORDS}
+
+*** Variables ***
+${SCRIPT}                     ${CURDIR}/nrf52840.resc
+${UART}                       sysbus.uart0
+${PROMPT}                     uart:~$
+
+*** Keywords ***
+Prepare Sample
+    [Arguments]                 ${sample_name}
+
+    Execute Command             $bin="${CURDIR}/artifacts/${sample_name}.elf"
+    Execute Script              ${SCRIPT}
+    Set Default Uart Timeout    60
```

Comment:
this is probably a lot. If we don't need explicit timeout, please drop this.

The value is in virtual time, and 60s virtual can take up to a few minutes realtime